### DISCUSSION TITLE
It's possible to filter "deprecated" charts but its' not possible to hide them in the chart selector

### DISCUSSION THREAD

**Discussion by xavier-GitHub76** [apache/superset#29712] [Ideas]
Hello,

It's possible to filter "deprecated" charts but its' not possible to hide them in the chart selector
![image](https://github.com/user-attachments/assets/430e2f0e-154a-4a44-b5e8-25fcad517a05)

Best regards

**Comment by sfirke** [comment]
You can restrict users from being able to see certain chart types by adding those charts to the `VIZ_TYPE_DENYLIST` in your config.  See https://github.com/apache/superset/blob/master/superset/config.py#L876.  Does that answer your question?

**Comment by xavier-GitHub76** [comment]
Thank you for your response 👍 
I discovered the VIZ_TYPE_DENYLIST option which can provide the service but imposes the choice on everyone.

I imagined more of a check button so that a user could show/hide obsolete graphics according to their preferences.
